js/gateio.js

```diff
@@ -326,6 +326,8 @@ module.exports = class gateio extends Exchange {
         let id = this.safeString (trade, 'tradeID');
         id = this.safeString (trade, 'id', id);
         let orderId = this.safeString (trade, 'orderid');
+        if (typeof orderId !== 'undefined')
+            orderId = this.safeString (trade, 'orderNumber');
```

Comment:
Hey ) Thx again, good to hear from you!

JFYI you can also do this like so:

```JavaScript
let orderId = this.safeString (trade, 'orderid');
// choose either of orderid or orderNumber
orderId = this.safeString (trade, 'orderNumber', orderId); // third value is the default
```

But no worries, your version is fine and user-friendly )